cmd/createdkg.go

```diff
@@ -77,7 +77,6 @@ func mustMarkFlagRequired(cmd *cobra.Command, flag string) {
 
 func runCreateDKG(ctx context.Context, conf createDKGConfig) (err error) {
 	// Map prater to goerli to ensure backwards compatibility with older cluster definitions.
-	// TODO(xenowits): Remove the mapping later.
```

Comment:
We can remove code for goerli entirely when goerli gets deprecated in a couple of weeks

https://blog.ethereum.org/2024/01/10/goerli-dencun-announcement